Sources/ProjectDescription/Config.swift

```diff
@@ -128,6 +132,10 @@ extension Config.GenerationOptions {
             try container.decode(Bool.self, forKey: .resolveDependenciesWithSystemScm)
         {
             self = .resolveDependenciesWithSystemScm
+        } else if container.allKeys.contains(.disableBundleAccessors),
```

Comment:
could you test this logic by adding this option [here](https://github.com/tuist/tuist/blob/main/Tests/ProjectDescriptionTests/ConfigTests.swift#L5)?

Reply:
I've already added this option in `ConfigTests.swift` (https://github.com/tuist/tuist/pull/3088/files#diff-98305cf446c3f2af6eae9e4746a79d6671dde48b998ff784b42cd822621d4c73) What else would you like to have there?

Reply:
ah, sorry, missed it, consider this as resolved then 🙂